modules/ROOT/pages/sap/sap-s4-hana-connector-reference.adoc

```diff
@@ -0,0 +1,518 @@
+| Default headers a| Array of <<defaultHeader>> |  +++Default HTTP headers the message should include.+++ |  | {nbsp}
+| Default query parameters a| Array of <<defaultQueryParam>> |  +++Default Query parameters the request should include.+++ |  | {nbsp}
+| Proxy configuration a| <<ProxyConfiguration>> |  +++Proxy configuration for the connector.+++ |  | {nbsp}
+| TLS configuration a| <<Tls>> |  +++Protocol to use for communication. Valid values are HTTP and HTTPS. Default value is HTTP. When using HTTPS the HTTP communication is going to be secured using TLS / SSL. If HTTPS was configured as protocol then the user needs to configure at least the keystore in the tls:context child element of this listener-config.+++ |  | {nbsp}
+| Reconnection a| <<Reconnection>> |  +++When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy+++ |  | {nbsp}
+|======================
+
+==== Associated Operations
+* <<createEntity>> {nbsp}
+* <<deleteEntity>> {nbsp}
+* <<executeFunction>> {nbsp}
+* <<getEntity>> {nbsp}
+* <<query>> {nbsp}
+* <<updateEntity>> {nbsp}
+
+
+
+== Operations
+
+[[createEntity]]
+=== Create entity
+`<s4hana:create-entity>`
+
++++
+Executes a create operation within the connected SAP S/4HANA instance. This is a dynamic operation, returning whatever the OData specification states that will be returned by the operation.
++++
+
+==== Parameters
+[cols=".^20%,.^20%,.^35%,.^20%,^.^5%", options="header"]
+|======================
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | *x*{nbsp}
+| Service a| String |  +++The type of service. This is defined from the services available in the SAP S/4HANA instance.+++ |  | *x*{nbsp}
+| Entity type a| String |  +++The type of entity over which the operation is executed. This is defined from the types of entities available in the selected service.+++ |  | *x*{nbsp}
+| Entity a| Object |  +++The entity to be affected in the SAP S/4HANA instance. This entity must be of the type defined in the type parameter.+++ |  +++#[payload]+++ | {nbsp}
+| Custom headers a| Object |  +++Custom headers that the request will include. The ones specified here will overwrite the default ones specified in the config.+++ |  | {nbsp}
+| Custom query parameters a| Object |  +++Custom query parameters that the request will include. The ones specified here will overwrite the default ones specified in the config.+++ |  | {nbsp}
+| Target Variable a| String |  +++The name of a variable on which the operation's output will be placed+++ |  | {nbsp}
+| Target Value a| String |  +++An expression that will be evaluated against the operation's output and the outcome of that expression will be stored in the target variable+++ |  +++#[payload]+++ | {nbsp}
```

Comment:
Please break into two sentences: 

An expression that will be evaluated against the operation's output. The outcome of that expression will be stored in the target variable.